_rules/aria-required-context-role-ff89c9.md

```diff
@@ -0,0 +1,229 @@
+- [Required Context Role](https://www.w3.org/TR/wai-aria-1.1/#scope)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+Element with [explicit semantic role](#explicit-role) `listitem` is contained within its [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) `list`, expressed as an [explicit semantic role](#explicit-role).
+
+```html
+<div role="list">
+	<div role="listitem">List item 1</div>
+</div>
+```
+
+#### Passed Example 2
+
+Element with [explicit semantic role](#explicit-role) `listitem` is contained within its [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) `list`, through the [implicit semantic role](#implicit-role) of `ul`.
+
+```html
+<ul>
+	<div role="listitem">List item 1</div>
+</ul>
+```
+
+#### Passed Example 3
+
+Element contained within its [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) even though it is not a direct child of the [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="list">
+	<div role="presentation">
+		<div role="listitem">List item 1</div>
+	</div>
+</div>
+```
+
+#### Passed Example 4
+
+`aria-owns` used to give the target element the right [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="list" aria-owns="id1"></div>
+<div id="id1" role="listitem">List item 1</div>
+```
+
+#### Passed Example 5
+
+`aria-owns` trumps ownership by closest ancestor, giving the element with [explicit semantic role](#explicit-role) of `listitem` the correct [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="list" aria-owns="id1">
+	<div role="navigation">
+		<div id="id1" role="listitem">List item 1</div>
+	</div>
+</div>
+```
+
+#### Passed Example 6
+
+Since implicit ownership can cross shadow boundaries, the element with the [explicit semantic role](#explicit-role) of `listitem` is contained within its [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) `list`.
+
+```html
+<div id="host" role="list"></div>
+
+<script>
+	const host = document.querySelector('#host')
+	const root = host.attachShadow({ mode: 'open' })
+	root.innerHTML = '<div role="listitem">List item 1</div>'
+</script>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The `listitem` has no [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="listitem">List item 1</div>
+```
+
+#### Failed Example 2
+
+The `listitem` is owned by the `tabpanel`, because it is the closest ancestor, but `tabpanel` is not the correct [context](https://www.w3.org/TR/wai-aria-1.1/#scope) for `listitem`.
+
+```html
+<div role="list">
+	<div role="tabpanel">
+		<div role="listitem">List item 1</div>
+	</div>
+</div>
+```
+
+#### Failed Example 3
+
+The `listitem` is owned by the `aria-label="menu"` div, rather than the `list`.
+
+```html
+<div role="list">
+	<div aria-label="menu">
+		<div role="listitem">List item 1</div>
+	</div>
+</div>
```

Comment:
I'm not fully understanding why we want this to fail rather than pass (see also comment on `owned-by`).

Reply:
Got better understanding after discussion. Thanks.